doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -43,7 +43,187 @@ All the JSON log types share a common structure:
 ::
 
 
-  {"timestamp":"2009-11-24T21:27:09.534255","event_type":"TYPE", ...tuple... ,"TYPE":{ ... type specific content ... }}
+  {"timestamp":"2009-11-24T21:27:09.534255","flow_id":ID_NUMBER, "event_type":"TYPE", ...tuple... ,"TYPE":{ ... type specific content ... }}
+
+Flow_id
```

Comment:
I know this was brought up in a previous version of this PR, but I think we should be exact, or use a proper title.  So `flow_id` or "Flow ID".